django-stubs/db/models/base.pyi

```diff
@@ -71,13 +71,15 @@ class Model(metaclass=ModelBase):
     def get_constraints(self) -> list[tuple[type[Model], Sequence[BaseConstraint]]]: ...
     def save(
         self,
+        *args: Never,
```

Comment:
Two things here...

First, I have only left the `*args` in to appease stubtest. We could perhaps ignore it instead, since the future is keyword-only, but then we might miss future keyword args.

Second, I picked `NoReturn` to stop typing users from passing the arguments. I think that's acceptable.

Third, [this ruff rule](https://docs.astral.sh/ruff/rules/no-return-argument-annotation-in-stub/) made me rewrite `NoReturn` as `Never`. But `Never` was only added in Python 3.11, while we still try to support down to 3.9. Is this okay? Does Mypy know the new name already on old Pythons, for stub files at least?

Reply:
Okay, tests answered my third question. Python 3.9 failed, so I reverted to `NoReturn` and disabled that particular Ruff rule.